endpoints/get.go

```diff
@@ -38,9 +46,16 @@ func NewGetHandler(storage backends.Backend, metrics *metrics.Metrics, allowCust
 
 func (e *GetHandler) handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	e.metrics.RecordGetTotal()
+
+	// If incoming request comes with a referer header, there's a e.cfg.refererLogRate percent chance
```

Comment:
IMHO: You don't need these comments. You added them in this commit, is that related to the `RandomPick` name?